composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/ResultItem.kt

```diff
@@ -2,6 +2,10 @@ package org.ooni.probe.data.models
 
 import kotlin.time.Duration.Companion.seconds
 
+data class ResultCount(val total: Long, val succeeded: Long) {
+    val failed get() = total - succeeded
```

Comment:
I just realised that we're mixing failed and anomalies here. 
Blocked = isAnomaly
Failed = isFailed

We shouldn't be counting failures inside the blocked count.